packages/samples/sources/programmability/dynamic-collections.move

```diff
@@ -56,32 +57,33 @@ public struct UserRegistry has key {
 // ANCHOR_END: table_struct
 
 // ANCHOR: table_usage
```

Comment:
A correct patch here would be to not add offsets / change update the code but to place the anchor after the `#[test] fun test_table() {` so that this line is not included in the final output. Good catch nevertheless!